srcdoc/configure/syntax_and_structure.mt

```diff
@@ -125,4 +125,39 @@ hosts:
 EOT
 ?>
 
+<h3 id="include_other_files">Using Inclusion</h3>
```

Comment:
Change to `Including Files`?

Sorry if my suggestion to add a _using_ section has led to this choice of wording.